### PR TITLE
Fix: Pass cli variables into scripts (fixes #3365)

### LINKED ISSUE #3365: Spoor scripts, hardcoded `course` folder
### Subject of the issue
It should be possible to use `grunt dev --coursedir=alternative` to specify a different name for `build/course`. The spoor script current isn't aware of this request.

https://github.com/adaptlearning/adapt-contrib-spoor/blob/3955b26fd4a1810cac98adfdc9c804440f718f40/scripts/postcopy.js#L3-L5

The `coursedir` variable isn't passed into scripts, so this is also not currently possible.

### PR DESCRIPTION
**Pull request by oliverfoster**
fixes #3365 

### Fix
* Add `coursedir` (and all other compliation variables) in script `options` arguments

### CODE REVIEW

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by swashbuck — APPROVED** [review]
👍

File: grunt/tasks/scripts.js [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
+    const taskCallback = this.async();
 
     if (options.plugins) {
 
-      var paths = [];
-      for (var i = 0, l = options.plugins.length; i < l; i++) {
-        var src = options.plugins[i];
+      let paths = [];
+      for (let i = 0, l = options.plugins.length; i < l; i++) {
+        const src = options.plugins[i];
```
Comment (danielghost) [inline grunt/tasks/scripts.js:21]:
```suggestion
      options.plugins.forEach(src => {
```

**Review by danielghost — CHANGES_REQUESTED** [review]
Very minor suggestion: https://github.com/adaptlearning/adapt_framework/pull/3366/files#r1129390620

**Pull request merged**